src/components/safe-apps/AppFrame/index.tsx

```diff
@@ -36,23 +36,18 @@ import useSignMessageModal from '../SignMessageModal/useSignMessageModal'
 import TransactionQueueBar, { TRANSACTION_BAR_HEIGHT } from './TransactionQueueBar'
 import PermissionsPrompt from '../PermissionsPrompt'
 import { PermissionStatus } from '../types'
+import AppIframe from './iframe'
 
 import css from './styles.module.css'
-import classnames from 'classnames'
 
 const UNKNOWN_APP_NAME = 'Unknown App'
 
-type AppFrameProps = {
+type AppFrameWrapperProps = {
   appUrl: string
   allowedFeaturesList: string
-  isWidget?: boolean
 }
 
-// see sandbox mdn docs for more details https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox
-const IFRAME_SANDBOX_ALLOWED_FEATURES =
-  'allow-scripts allow-same-origin allow-popups allow-popups-to-escape-sandbox allow-forms allow-downloads allow-orientation-lock'
-
-const AppFrame = ({ appUrl, allowedFeaturesList, isWidget = false }: AppFrameProps): ReactElement => {
+const AppFrameWrapper = ({ appUrl, allowedFeaturesList }: AppFrameWrapperProps): ReactElement => {
```

Comment:
I think this component can continue being `AppFrame` as the folder name and the extracted iframe component itself could be `Iframe` with capital i. This components is more than a simple wrapper.

Now we have this `index.tsx` containing `AppFrameWrapper` and the `iframe.tsx` containing `AppIframe`, then this latest one is imported as `AppFrame` or `AppIframe` depending on where so for me is a little confusing.

Reply:
Good point. The only problem is `AppIframe` and `AppFrame` – maybe I should call `AppIframe` something else? Like `SafeAppIframe`.